src/main/webui/src/app/components/content/group/GroupList.jsx

```diff
@@ -14,8 +14,109 @@
+import {ListJsonDebugger} from "../common/Debugger.jsx";
+import ListControl from "../common/ListControl.jsx";
+import {groupOptionLegend as options} from "../../ComponentConstants.js";
+import {StoreListingWidget} from "../common/StoreListingWidget.jsx";
+import {LoadingSpiner} from "../common/LoadingSpiner.jsx";
+import {Utils} from "#utils/AppUtils.js";
+import {IndyRest} from "#utils/RestClient.js";
 
-export default function GruopList() {
-  return <div>This is not implemented yet!</div>;
+const {storeRes, disableRes} = IndyRest;
+
+const handlers = {
+  handleDebug: (event, setState) => {
+    setState({
+      enableDebug: event.target.checked,
+    });
+  },
+  handleSearch: (event, rawList, setState) => {
+    setState({
+      rawList,
+      listing: Utils.searchByKeyForNewStores(event.target.value, rawList),
+    });
+  },
+  handleSortBy: (event, rawList, setState) => {
+    setState({
+      rawList,
+      listing: Utils.sortByPropForStores(event.target.value, rawList),
+    });
+  },
+};
+
+export default function GroupList() {
+  const {packageType} = useParams();
+  const [state, setState] = useState({
+    rawList: [],
+    listing: [],
+    disabledMap: {},
+    enableDebug: false,
+    message: "",
+  });
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(true);
+    const fetchdData = async () => {
+      const res = await storeRes.getStores(packageType, "group");
+      if (res.success) {
+        const timeoutRes = await disableRes.getAllStoreTimeout();
+        let disabledMap = {};
+        if (timeoutRes.success) {
+          const timeoutData = timeoutRes.result;
+          disabledMap = Utils.setDisableMap(timeoutData);
+        } else {
+          Utils.logMessage(`disable timeout get failed in group listing! Error reason: ${timeoutRes.error.message}`,);
+        }
+        let data = res.result;
+        if (typeof data === "string") {
+          data = JSON.parse(data);
+        }
+        setState({
+          rawList: data.items,
+          listing: data.items,
+          disabledMap,
+        });
+      } else {
+        setState({
+          message: res.error.message,
+        });
+      }
+      setLoading(false);
+    };
+    fetchdData();
+  }, [packageType]);
+
+  if (loading) {
+    return <LoadingSpiner />;
+  }
+
+  return (
+    <React.Fragment>
+      <ListControl
+        type="group"
+        legends={options}
+        handleSearch={event => handlers.handleSearch(event, state.rawList, setState)
+        }
+        handleDebug={event => handlers.handleDebug(event, setState)}
+        handleSortBy={event => handlers.handleSortBy(event, state.rawList, setState)
+        }
+      />
+      {state.listing ?
+        <StoreListingWidget
+          storeList={state.listing}
+          disableMap={state.disabledMap}
+          storeType="group"
+        />
+       :
+        <div className="container-fluid">No content fetched!</div>
+      }
```

Comment:
I think this page lost the group constituents part. If you check the current UI page for group listing.

Reply:
added constituents section